runtime/src/main/java/org/corfudb/runtime/collections/StreamManager.java

```diff
@@ -123,6 +124,35 @@ void subscribe(@Nonnull StreamListener streamListener, @Nonnull String namespace
         subscriptions.put(streamListener, new StreamSubscriber(task, scheduledFuture));
     }
 
+    /**
+     * Subscribe to updates.
+     *
+     * @param streamListener   client listener
+     * @param namespace        namespace of interest
+     * @param streamTag        only updates of tables with the stream tag will be polled
+     * @param tablesOfInterest only updates from these tables will be returned
+     * @param startAddress     address to start the notifications from
```

Comment:
Seek moves the pointer to this address, if it's trimmed, then in next polling cycle, the poller will get a TrimmedException, stop the subscription and restart. This part is not changed since initially implemented.